ui/src/layout/AssetScans/AssetScansTable.js

```diff
@@ -3,12 +3,19 @@ import TablePage from 'components/TablePage';
 import { OPERATORS } from 'components/Filter';
 import { APIS } from 'utils/systemConsts';
 import { getFindingsColumnsConfigList, getVulnerabilitiesColumnConfigItem, formatDate, getAssetColumnsFiltersConfig,
-    findingsColumnsFiltersConfig, vulnerabilitiesCountersColumnsFiltersConfig, scanColumnsFiltersConfig } from 'utils/utils';
+    findingsColumnsFiltersConfig, vulnerabilitiesCountersColumnsFiltersConfig, scanColumnsFiltersConfig, getAssetName } from 'utils/utils';
 import { FILTER_TYPES } from 'context/FiltersProvider';
 import StatusIndicator, { STATUS_MAPPING } from './StatusIndicator';
 
 const TABLE_TITLE = "asset scans";
 
+const NAME_SORT_IDS = [
+    "asset.assetInfo.instanceID",
+    "asset.assetInfo.podName",
+    "asset.assetInfo.dirName",
+    "asset.assetInfo.name",
+    "asset.assetInfo.containerName"
+];
```

Comment:
I think we want imageID here too IIRC?

Reply:
You're right, good catch. I think `name` was renamed to `imageID` recently for the `ContainerImageInfo` type...